apps/front/components/AppHeader.vue

```diff
@@ -27,11 +27,14 @@
           <li><NuxtLink to="/">News</NuxtLink></li>
```

Comment:
<li><NuxtLink to="/">Home</NuxtLink></li>
          <li><NuxtLink to="/cryptocurrencies">Crypto</NuxtLink></li>
          <li><NuxtLink to="/news">News</NuxtLink></li>